### PR TITLE
Fix csoftmax init bug

### PR DESCRIPTION
**Pull request by Function-Samuel**
### PR Category
Communication Library

### PR Types
Bug fixes

### Description
c_cross_entropy_with_softmax算子迁移时遇到的初始化问题修复
PCard-85979

### CODE REVIEW

File: paddle/fluid/framework/new_executor/instruction/instruction_util.cc [hunk 1]
```diff
@@ -153,7 +181,9 @@ phi::DeviceContext* ParseDeviceContext(pir::Operation* op,
             op_name.compare(paddle::dialect::AllReduce_Op::name()) == 0 ||
             op_name.compare(paddle::dialect::Broadcast_Op::name()) == 0 ||
             op_name.compare(paddle::dialect::BroadcastOp::name()) == 0 ||
-            op_name.compare(paddle::dialect::AllGatherOp::name()) == 0) {
+            op_name.compare(paddle::dialect::AllGatherOp::name()) == 0 ||
+            op_name.compare(
+                paddle::dialect::CSoftmaxWithCrossEntropyOp::name()) == 0) {
```
Comment (liym27) [inline paddle/fluid/framework/new_executor/instruction/instruction_util.cc:172]: Delete vlog and comments.

**Review by liym27 — APPROVED** [review]
LGTM

**Comment by Function-Samuel** [comment]
> LGTM

thx😊

**Review by zhangbo9674 — APPROVED** [review]
LGTM

**Comment by Function-Samuel** [comment]
> LGTM

my pleasure😄

File: paddle/fluid/framework/new_executor/instruction/instruction_util.cc (hunk 1 above)
Comment (Function-Samuel) [reply paddle/fluid/framework/new_executor/instruction/instruction_util.cc:172]: > Delete vlog and comments.

done.

**Pull request merged**